packages/charts/react-charts-preview/library/src/components/LineChart/LineChart.tsx

```diff
@@ -184,7 +184,6 @@ export const LineChart: React.FunctionComponent<LineChartProps> = React.forwardR
     }, [props.height, props.width, props.data]);
 
     function _injectIndexPropertyInLineChartData(lineChartData?: LineChartPoints[]): LineChartDataWithIndex[] | [] {
-      const { allowMultipleShapesForPoints = false } = props;
```

Comment:
what is this change? Not using enable multiple shape prop anymore?

Reply:
allowMultipleShapesForPoints  is used as it is in _createLegends,_getBoxWidthOfShape,_getPath,_getPointFill functions. Within _injectIndexPropertyInLineChartData function, the index of the selection was earlier passed based on allowMultipleShapesForPoints prop. If allowMultipleShapesForPoints was true, index was passed as -1 which was causing the crash at _createLines function where it was getting index of point as -1.